### DISCUSSION TITLE
What is the motivation behind this project?

### DISCUSSION THREAD

**Discussion by musi00** [eclipse-edc/Connector#331] [Q&A] [answered]
Hello,

I'm interested in IDS-RAM and have been mostly following the official implementation of the dataspace connector hosted here: https://github.com/International-Data-Spaces-Association/DataspaceConnector

However I became aware of this repo recently but I don't understand the reason behind it.  How will this be different than the official implementation?  From what I read it is trying to merge the IDSA offcial implementation and the Microsoft implementation here: https://github.com/microsoft/Data-Appliance-GX.  But why?  What does the Microsoft implementation provide? The documentation on the Microsoft project is lacking.  Thanks for your time.

Mustafa

**Comment by mspiekermann** [comment, marked as answer]
Hi @musi00,

there is an issue with the page ranking in search engines actually. What you are refering to is the proposal, we submitted to become an Eclipse Foundation project, which has links to repositories that hold the code for our **initial commit** (a combination of Data Appliance GX and Fraunhofer's Dataspace Connector).  The project website can be found here https://projects.eclipse.org/projects/technology.dataspaceconnector which also hold all the information for the Developers Resources. 

**Since June **we are working in this repository** only** and the old ones will become archieved. 

So to follow the idea of the EDC and its implementation towards a dataspace platform compliant to IDS and Gaia-X, please follow (and gladly contribute to) our community. 

Markus

P.S. We asap will add more content to the project's website to get it ranked better in the search engines.
